pkg/virt-config/feature-gates.go

```diff
@@ -72,6 +72,8 @@ const (
 	VMLiveUpdateFeaturesGate = "VMLiveUpdateFeatures"
 	// When BochsDisplayForEFIGuests is enabled, EFI guests will be started with Bochs display instead of VGA
 	BochsDisplayForEFIGuests = "BochsDisplayForEFIGuests"
+	// BindingPlugingsGate enables using a plugin to bind the pod and the VM network
+	BindingPlugingsGate = "BindingPlugins"
```

Comment:
Can you add a `Network` prefix? It is not clear what binding is referred here.

Reply:
done.